gradle/libs.versions.toml

```diff
@@ -61,6 +61,7 @@ springCore = "5.3.37"
 svnkit = "1.10.11"
 sw360Client = "17.0.1-m2"
 wiremock = "3.8.0"
+xmlutil = "0.90.1"
```

Comment:
How does xmlutil compare to jackson for xml ? Is it similar powerful?

Reply:
Can you be a bit more specific about what kind of features you have in mind?